mission_list.md

```diff
@@ -0,0 +1,60 @@
+>
+>It is divided into two sections, solely to keep the requirements of each section clearly defined; Core and dApp.
+>It can be updated through an on-chain 2/3-majority rank-weighted vote across all Members.
+>
+>Some obvious cornerstones that must be upheld by all projects are:
+>
+>- Must be FOSS (Free and Open-Source Software).
+>- Source code must be accessible without a login requirement.
+>- All members of PoToC must have permission to propose changes and raise issues.
+>- The Mission List will be defined as part of the initial seeding.
+>
+>Core Tooling
+>
+>Generally, all tools that are used by The Fellowship to craft, test, and validate updates fall into this category. But only the tools that are predominantly developed for this cause shall be maintained by PoToC. For example, the Rust language is not predominantly developed for Polkadot; hence, it does not fit here.
+>
+>dApp Tooling
+>
+>This category contains all tools and libraries that are used to build on Polkadot and are predominantly developed for this cause.
+
+(This is my personal opinion on a 
+
+## DApp Tooling
+
+- [Polkadot-JS](https://github.com/polkadot-js/apps)
+- [PAPI](https://www.npmjs.com/package/@polkadot-api/substrate-client)
+- [Chopsticks](https://github.com/AcalaNetwork/chopsticks)
+- [Zombienet](https://github.com/paritytech/zombienet) + [v2](https://github.com/paritytech/zombienet-sdk)
+- [Substrate Connect](https://github.com/paritytech/substrate-connect)
+- [Scale TS](https://npmjs.com/package/scale-ts)
+
+## Core Tooling
+
+- [SubXt](https://github.com/paritytech/subxt)
+- [Subway](https://github.com/AcalaNetwork/subway)
+- [Zepter](https://github.com/ggwpez/zepter)
+- [Substrate Weight Compare](https://github.com/ggwpez/substrate-weight-compare)
+- [OpenGov CLI](https://github.com/joepetrowski/opengov-cli)
+- [Sidecar](https://github.com/paritytech/substrate-api-sidecar)
+- [scale-*](https://github.com/paritytech/scale-decode), [more](https://github.com/polkadot-tooling-collective/collective/blob/dd96bc056dfa47f0bc6a885c7afd25ed78345a11/join_request/jameswilson.yml#L26)
+- [srtool](https://github.com/paritytech/srtool) (maintainer did not apply yet)
+- [subwasm](https://github.com/chevdor/subwasm) (ditto)
+- [virto SDK/libwallet](https://github.com/virto-network/virto-sdk/tree/main/libwallet)
+- [virto SDK/scales](https://github.com/virto-network/virto-sdk/tree/main/scales)
+
+## Undecided
+
+- [virto SDK/sube](https://github.com/virto-network/virto-sdk/tree/main/sube)
+- [polkadot-JS extension](https://github.com/polkadot-js/extension)
+- [cypress polkadot wallet](https://github.com/ChainSafe/cypress-polkadot-wallet)
+- [Zondax Ledger](https://github.com/zondax/ledger-polkadot)
+- [Rust Test](https://github.com/ltfschoen/RustTest)
+- [Polkadot JS test](https://github.com/ltfschoen/PolkadotJSTest)
+- [NFT coretime](https://github.com/ltfschoen/nft-coretime)
+- [Flappy tips](https://github.com/ltfschoen/flappytips)
```

Comment:
This looks like some game? Not a developer tool?

Reply:
the goal is to make it an end-to-end browser based Web3 **game developer tool** that integrates with game engines like p5.js, the Polkadot.js Extension, and ink! smart contracts.
 
At the moment it shows how to create a multiplayer browser based game using a traditional browser-based game engine like p5.js that integrates with Polkadot.js API (it request tips at the end of the game from the treasury based on your results) through to production, but it's not adequately decentralized at the moment, so the next step was make more of the gameplay progressively decentralized and to finish integrating the leaderboard ink! smart contract. 
i was also thinking of adding a source of randomness so it couldn't be gamed, initially i was going to use a future block hash for the source of randomness (see my XCMTemplate repo) since using randomness precompiles weren't adequate, but then i investigated using ZK as the source of randomness and studied ZK for about 6 months last year, but recently someone suggested I could use a Phala secure enclave for the source of randomness.

Reply:
if you see value in decoupling only the specific Web3 game developer tooling related aspects so we can use it in a tooling library please let me know.

Reply:
that repo has actual won two hackathons.

it got 2nd Place Visualisation Bounty Challenge for Build Polkadot: Network
Launch hackathon in 2020 https://polkadot.network/blog/build-polkadot-network-launch-bounty-challenges#visualization

and then i improved upon it in a subsequent hackathon doing things like making it multiplayer instead of single player and won Zeitgeist's Ecosystem Auxiliary application Bounty Sponsor Challenge for the Polkadot Hackathon: Europe Edition in 2023

Reply:
Every project that we include in this list here is a promise from PoToC to maintain it. For now i dont see a good point for PoToC in doing so. If it evolved into a gaming DApp library or something similar than that would make a good point. But until then, it is not a developer tool but "a cute game that allows you to fly the DOT character through block gaps" as the hackathon stated.